code-challenges.test.js

```diff
@@ -21,9 +21,36 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// describe("fibonacciMaker", () => {
+//   it("returns an array of numbers of the fibonacchi sequence at the length of that number", () => {
+//     expect(fibonacciMaker("some number")).toEqual("fibonacci sequence array at the number length")
 
 // b) Create the function that makes the test pass.
-
+// Pseudo Code:
+// declare a function
+// set the parameter to a number
+// research finboacchi sequence -- the next number is found by adding up the two numbers before it
+// [num 1, num2, num3 = num1 + num2, num4 = num2 + num3]
+// method to isolate the value of each index
+// create a new array at the length of the number input
+// start with first 2 numbers as 1, then value will be previous 2 numbers added together
+// itterate through array
+// return array of fibonacci numbers
+
+let array = [1,1]
+const fibonacciMaker = (number) =>   {
+   for (let i=2; i<number; i++){
+     array[i] = array[i-2] + array[i-1]
+   }
+   return array
+}
+
```

Comment:
This is wild that the test still runs below the function. Best practices are not only for flow but to help another developer understand our code. In the test suite the flow is to develop the test first then follow it with out desired function. Other than that great approach. The code is clean and logical.

Reply:
Noted that test should be before function, thank you!